### DISCUSSION TITLE
V3: What are the alternatives to the `discoverUninitializedComponents` method?

### DISCUSSION THREAD

**Discussion by osbre** [alpinejs/alpine#2090] [1. Help] [answered]
There is a package that depends on that method:

https://github.com/SimoTod/alpine-turbo-drive-adapter/blob/9b0bd15af763a0cae69ec4841b7ccaf2f039ebf7/src/bridge.js#L30-33

Any feedback on how it can be replaced?

Method source in `2.x` branch:
https://github.com/alpinejs/alpine/blob/b5bf802934e981e851cfb3d3797b49379a736969/src/index.js#L45

Thanks

**Comment by SimoTod** [comment]
It's worth noting that the package you mentioned is for alpine v2.

**Comment by osbre** [comment, marked as answer]
The package for Turbolinks support got updated, which makes this question no longer active.
https://github.com/SimoTod/alpine-turbo-drive-adapter
